Makefile.pc.mk

```diff
@@ -1,7 +1,31 @@
-CPP_PC := gcc
-CPP_FLAGS_PC := -w -fsanitize=address -DVERSION_PC -DPERMUTER
+TARGET              := sound
+PC_BUILD_DIR	    := build/pc
 
-C_FILES_PC := $(SRC_DIR)/pc/main.c $(SRC_DIR)/dra/91EBC.c $(SRC_DIR)/dra/92F60.c $(SRC_DIR)/dra/93290.c $(SRC_DIR)/dra/93BDC.c $(SRC_DIR)/dra/94F50.c $(SRC_DIR)/dra/953A0.c
+CC_PC               := gcc
+LD_PC               := ld
+CC_FLAGS_PC         := -c -g -DVERSION_PC -DPERMUTER  -fno-stack-protector 
```

Comment:
-fsanitize=address probably shouldn't be on by default like it was, but just figure I'd point out it's pretty useful